movieapp/src/App.tsx

```diff
@@ -3,47 +3,47 @@ import { Footer } from './containers/Footer/Footer';
 import { HomePage } from './pages/HomePage/HomePage';
 import './services/i18n';
 import { useState, useEffect } from 'react';
-
-type Movie = {
-  [id: string]: string;
-  title: string;
-  description: string;
-};
+import { Movie } from './models/types';
 
 const moviesMock: Array<Movie> = [
   {
     id: 'm1',
     title: 'Movie 1',
-    description: 'Description for Movie 1'
+    description: 'Description for Movie 1',
+    fullDescription: 'Full description for Movie 1'
   },
   {
     id: 'm2',
     title: 'Movie 2',
-    description: 'Description for Movie 2'
+    description: 'Description for Movie 2',
+    fullDescription: 'Full description for Movie 2'
   },
   {
     id: 'm3',
     title: 'Movie 3',
-    description: 'Description for Movie 3'
+    description: 'Description for Movie 3',
+    fullDescription: 'Full description for Movie 3'
   },
   {
     id: 'm4',
     title: 'Movie 4',
-    description: 'Description for Movie 4'
+    description: 'Description for Movie 4',
+    fullDescription: 'Full description for Movie 4'
   }
 ];
 
 export function App(): JSX.Element {
   const [movies, setMovies] = useState<Movie[]>([]);
+  const [expandedMovie, setExpandedMovie]: any = useState();
```

Comment:
why do u use here `any`? u should add type of state https://www.robinwieruch.de/typescript-react-usestate/

Reply:
fixed